bundles/org.openhab.core/src/main/java/org/openhab/core/service/StartLevelService.java

```diff
@@ -75,6 +75,8 @@ public class StartLevelService {
 
     public static final String STARTLEVEL_MARKER_TYPE = "startlevel";
 
+    public static final String STARTLEVELSERVICE_SOURCE = "org.openhab.core.service.StartLevelService";
```

Comment:
The same as above

Reply:
Thanks for the hint. I've just changed that.